src/Map.jsx

```diff
@@ -87,11 +101,22 @@ class Map extends Component {
       stops = [];
     }
     if (stops.length === 0
-      || (stops.length === 1 && stops[0] !== stopCoordinates)) {
+      || (stops.length === 1 && !this.checkIfTwoStopsAreEqual(stops[0], stopCoordinates))) {
       console.log(`Stop Sid: ${stopInfo.sid}`);
       stops.push(stopInfo);
     }
     this.setState({ selectedStops: stops });
+    if (stops.length === 2) {
+      const routeSegment = this.getRouteBetweenStops(route);
+      console.log(routeSegment);
+    }
+  }
+  /**
+   * sees if two stops are equal by evaluating their coordinates
+   */
+  checkIfTwoStopsAreEqual(stop1, stop2) {
```

Comment:
@EddyIonescu fixes bug where I was inaccurately comparing two stops to see if they're equal
We only want to keep track of two unique stops in case the user clicks the same one twice

Reply:
yes, sounds reasonable